ResumeBuilderApp/forms.py

```diff
@@ -7,3 +7,8 @@ class CreateUserForm(UserCreationForm):
     class Meta:
         model = User
         fields = ['email', 'password1', 'password2']
+
+class UserEditForm(forms.ModelForm): #comment so I can re-push
+  class Meta:
+      model = User
+      fields = ['first_name', 'last_name', 'email', 'phone_number', 'address', 'is_staff', 'is_active']
```

Comment:
I believe we will need to add phone_number and address to models.py because it's not in Django's user model